lib/net.js

```diff
@@ -1384,6 +1384,14 @@ Server.prototype.listen = function() {
   return self;
 };
 
+Object.defineProperty(Server.prototype, 'listening', {
+  get: function() {
+    return !!this._handle;
+  },
+  configurable: false,
```

Comment:
Why `configurable: false`? This means that user-land won't be able to re-define this property for any reason. Looking at the source, other areas seem to have `configurable: true`, like the other `Object.defineProperty` in this file as well as the ones in https://github.com/nodejs/node/blob/6fff47ffacfe663efeb0d31ebd700a65bf5521ba/lib/internal/streams/lazy_transform.js and https://github.com/nodejs/node/blob/a5cce79ec3f0228dd3ff4311022efeb72df21426/lib/console.js